### ISSUE TITLE
Game crash after grabbing my B.O.B during intro.

### ISSUE THREAD

**Issue by meatfaceflyingdragon** [Zurek0x/ZombiU_XPatcher#3]
Moved all files from contents\version1.4 to the game directory, ran patch, and entered game directory. However, my game crashes whenever I try to grab my B.O.B during the tutorial.

**Comment by tqoac** [comment]
Switch ubisoft connect / game launcher to offline mode.

**Comment by mrchrismania** [comment]
did you have any success in fixing it? im curious myself

**Comment by meatfaceflyingdragon** [comment]
> did you have any success in fixing it? im curious myself

Nope, still don't really know what was up. Ended up playing the game in vanilla

**Comment by mrchrismania** [comment]
> > did you have any success in fixing it? im curious myself
> 
> Nope, still don't really know what was up. Ended up playing the game in vanilla


i had the ubisoft connect version so i ended up just buying the steam version installed the patch and it worked

**Comment by Pipesin** [comment]
I have the same problem and apparently there is still no other solution other than buying the Steam version and installing the patch on that platform.
